extensions/notebook/resources/notebooks/JupyterBooksCreate.ipynb

```diff
@@ -41,7 +41,7 @@
                 "#install jupyter-book\r\n",
                 "cmd = f'{sys.executable} -m pip show jupyter-book'\r\n",
                 "cmdOutput = !{cmd}\r\n",
-                "if len(cmdOutput) > 0 and '0.6.4' in cmdOutput[1]:\r\n",
+                "if len(cmdOutput) > 1 and '0.6.4' in cmdOutput[1]:\r\n",
```

Comment:
isn't this already in master (and the release branch)?

Reply:
just merged mater.